src/main/java/org/openpnp/logging/SystemLogger.java

```diff
@@ -42,5 +38,34 @@ public void write(byte[] buf, int off, int len) {
                 Logger.error(str);
                 break;
         }
+
+    }
+
+    @Override
+    public void write(byte[] buf, int off, int len) {
+        /*
+         * Log to tinylog as long as the Logger is available (before shutdown)
+         * As tinylog will forward all logs to the console, we do not call the super method
+         */
+        if (logToLogger) {
+            byte[] pb = new byte[len];
```

Comment:
you are right, i did not think it over after refactoring (where it came from), will change it